roles/base_patch/tasks/main.yml

```diff
@@ -17,3 +17,16 @@
   become_user: "{{ remote_admin_account }}"
   become: yes
   shell: yum update-minimal --security -y
+
+- name: "add amazon ntp server to chrony.conf"
+  lineinfile:
+    path: /etc/chrony.conf
+    insertafter: EOF
+    line: 'server 169.254.169.123 prefer iburst'
```

Comment:
Wait, it's a 169 address? Not that I disbelieve you, but that's weird.

Reply:
Yep, from Amazon's docs:

> The Amazon Time Sync Service is available through NTP at the 169.254.169.123 IP address for any instance running in a VPC. Your instance does not require access to the internet, and you do not have to configure your security group rules or your network ACL rules to allow access.